--- DbConnection.php    2016-03-22 13:02:15.000000000 +0000
+++ DbConnection.php_new        2016-03-23 09:44:57.154254424 +0000
@@ -152,11 +152,12 @@
                  */
                 $driverOptions[PDO::MYSQL_ATTR_INIT_COMMAND] =
                     'SET SESSION SQL_MODE=\'STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,'
-                    . 'NO_AUTO_CREATE_USER,ANSI_QUOTES,PIPES_AS_CONCAT,NO_ENGINE_SUBSTITUTION\';';
+                    . 'NO_AUTO_CREATE_USER,ANSI_QUOTES,PIPES_AS_CONCAT,NO_ENGINE_SUBSTITUTION\'';
                 if (array_key_exists('charset', $adapterParamaters) && $adapterParamaters['charset']) {
-                    $driverOptions[PDO::MYSQL_ATTR_INIT_COMMAND] .= 'SET NAMES ' . $adapterParamaters['charset']. ';';
+                    $driverOptions[PDO::MYSQL_ATTR_INIT_COMMAND] .= ', NAMES ' . $adapterParamaters['charset'];
                     unset($adapterParamaters['charset']);
                 }
+                $driverOptions[PDO::MYSQL_ATTR_INIT_COMMAND] .=';';

                 $adapterParamaters['port'] = $this->config->get('port', 3306);
                 break;
